_plugins/style_guide_assets.rb

```diff
@@ -5,28 +5,22 @@
 
 require 'fileutils'
 
-def copy_assets(origin, destination)
-  origin_glob = File.join(origin, '**/*')
-  origin_files = Dir.glob(origin_glob).reject { |f| File.directory?(f) }
+DESIGN_SYSTEM_ASSETS_TO_COPY = %w[
+  fonts
+  img
+  js/init.js
+]
```

Comment:
This reminds me that in other sites we use a different variation of this kind of plugin which copies files and is configured through `_config.yml`, wonder if that might be possible to use here. Would be nice to use a consistent approach.

https://github.com/GSA-TTS/identity-handbook/blob/3f16eeb/_config.yml#L20-L21

https://github.com/GSA-TTS/identity-handbook/blob/main/_plugins/copy_to_destination.rb

Reply:
yeah that's where I must have remembered from

Reply:
So it wasn't as straightforward to port over as-is since we don't want _all_ assets folders, and for some folders we only want specific files. But I came up with a solution in 0b7b4fa that I think is pretty elegant and also something we could port back to the other projects we use if we want it to be consistent.

Reply:
Testing to see if that'll work as something like this, which would be nice to support:

```
copy_files:
  from: foo
  to: bar
```

Reply:
> Testing to see if that'll work as something like this, which would be nice to support:

Made supportable in 7ff94da.